kong/constants.lua

```diff
@@ -140,7 +140,7 @@ local constants = {
   },
   REPORTS = {
     ADDRESS = "kong-hf.konghq.com",
-    STATS_PORT = 61830
+    STATS_PORT = 61833,
```

Comment:
```suggestion
    STATS_PORT_TLS = 61833,
```

So it is more obvious this is a TLS port.